src/backend/optimizer/plan/planner.c

```diff
@@ -1220,12 +1220,6 @@ inheritance_planner(PlannerInfo *root)
 			}
 		}
 
-		/**
-		 * The grouping planner scribbles on the rtable e.g. to add pseudo columns.
-		 * We need to keep track of this.
-		 */
-		parse->rtable = subroot.parse->rtable;
```

Comment:
could you provide more information on why it's safe to remove this line?

Reply:
It will be in final_rtable. Besides, the assignment seems to not correct.